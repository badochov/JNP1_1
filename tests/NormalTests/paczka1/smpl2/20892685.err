Error in line 6: np3z S368 22,4
Error in line 9: OWgsAu3H S409 95,6
Error in line 15: 2kJ S236 62,5
Error in line 22: A702 S368 81,7
Error in line 14: np3z S409 73,5
