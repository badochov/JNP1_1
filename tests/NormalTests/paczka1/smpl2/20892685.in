L3X3sn S236 57,9
OWgsAu3H S236 88,7
A702 S368 40,4
L3X3sn S236 250,1
A702 S368 150,7
np3z S368 22,4
OWgsAu3H S236 136,6

OWgsAu3H S409 95,6
np3z S409 39,8
OWgsAu3H A702 68,0
np3z S409 50,7
A702 S236 6,5
np3z S409 73,5
2kJ S236 62,5
txzNjwW6Lb S236 94,8
?
A702 S236 9,3
?S236
2kJ A702 49,3
?
A702 S368 81,7
2kJ A702 18,2
2kJ A702 88,3
CPMXnDb S236 60,7
OWgsAu3H A702 333,2
2kJ A702 431,8
?
?S236
txzNjwW6Lb S236 36,8
A702 A702 92,9
np3z S236 55,0

np3z S236 127,6
2kJ S236 80,4
?txzNjwW6Lb
txzNjwW6Lb S368 10,3
txzNjwW6Lb S368 27,3

A702 A702 189,5