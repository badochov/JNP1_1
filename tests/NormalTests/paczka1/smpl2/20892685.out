A702 S 110,3
L3X3sn S 192,2
OWgsAu3H S 47,9
np3z S 10,9
S236 240,1
S368 110,3
S409 10,9
S236 242,9
A702 S 113,1
L3X3sn S 192,2
OWgsAu3H S 47,9
np3z S 10,9
S236 242,9
S368 110,3
S409 10,9
2kJ A 374,6
A702 S 113,1
L3X3sn S 192,2
OWgsAu3H A 265,2 S 47,9
np3z S 10,9
S236 242,9
S368 110,3
S409 10,9
A702 639,8
S236 242,9
txzNjwW6Lb S 58,0
